gradle/guide-build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
     }
 
     dependencies {
-        classpath "org.grails:grails-docs:6.0.0-RC1"
+        classpath "org.grails:grails-docs:6.2.0"
```

Comment:
Upgrading `grails-docs` is not necessary for the fix to work. I just figured I might as well update it to the latest version.